packages/test/test-end-to-end-tests/src/test/directoryEndToEndTests.spec.ts

```diff
@@ -992,13 +1005,20 @@ describeNoCompat("SharedDirectory orderSequentially", (getTestObjectProvider) =>
         assert.equal(subDirDeletedEventData.length, 0);
         // rollback
         assert.equal(subDirCreatedEventData.length, 0);
+        assert.equal(undisposedEventData.length, 0);
```

Comment:
Since the events have to be registered on the subdirectories themselves, is there a point to leaving this here? Can I check that the event doesn't get fired some other way?

Reply:
Yeah, this check doesn't even make sense since there is nothing to subscribe to. I don't think it's particularly valuable, but if you wanted, you could check in any other test that rolls back other subdirectory operations besides delete, like "Should rollback newly created subdirectory"